@here/harp-datasource-protocol/lib/Pixels.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const RX = /^((?=\.\d|\d)(?:\d+)?(?:\.?\d*))px$/;
+
+/**
+ * A class representing Pixels.
+ *
+ * @hidden
+ * @internal
+ */
+export class Pixels {
+    /**
+     * Parses a pixel string literal.
+     *
+     * @param text The string color literal
+     */
+    static parse(text: string): Pixels | undefined {
+        const match = RX.exec(text);
+        if (match === null) {
+            return undefined;
+        }
+        return new Pixels(Number(match[1]));
+    }
+
+    /**
+     * Constructs a [[Pixels]] literal
+     *
+     * @param value The number of pixels.
+     */
+    constructor(readonly value: number) {}
+
+    toJSON() {
+        // tslint:disable-next-line: no-bitwise
+        return `${this.value >> 0}px`;
```

Comment:
Why casting as integer ?
We support floating points at parsing, why loosing information here ?

Reply:
fixed